app/Services/Email.php

```diff
@@ -79,6 +79,7 @@ protected function defineTokens($user)
             ':reportback_noun:'       => strtolower($this->contest->campaign->reportback_info->noun),
             ':reportback_verb:'       => strtolower($this->contest->campaign->reportback_info->verb),
             ':sender_name:'           => $this->contest->sender_name,
+            ':rules:'                 => ! is_null($this->competition) ? $this->competition->rules : '',
```

Comment:
Maybe we should call this token `:rules_url:` or something like that. So admins can be clear if they just want to print out the raw URL in the email or if they want to be able to hyperlink with in text with markdown. Similar to how we do with the `prove_it_link`